lib/mutant/meta/example/verification.rb

```diff
@@ -0,0 +1,86 @@
+module Mutant
+  module Meta
+    class Example
+      # Example verification
+      class Verification
+        include Adamantium::Flat, Concord.new(:example, :mutations)
+
+        # Test if mutation was verified successfully
+        #
+        # @return [Boolean]
+        def success?
+          missing.empty? && unexpected.empty? && no_diffs.empty?
+        end
+
+        # Error report
+        #
+        # @return [String]
+        def error_report
+          fail 'no error report on successful validation' if success?
```

Comment:
Its a `RuntimeError`, and I do not want anyone to consider rescuing these errors, so I do not want to give any specific hint to do so via a dedicated exception class. This is only used from specs, and they *should* bubble to a failed example.